### DISCUSSION TITLE
New URL Feedback: Please Vote

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#18816] [Other Feature Feedback, Questions, & Ideas]
## Feedback needed ✅  

[Please vote for your preferred URL on this anonymous survey](https://survey3.medallia.com/?ny91lt-url-pref). 
The survey will close on June 23rd @ 12:00pm PST.
*It's completely anonymous  - please only vote once.

**Comment by doguedogue** [comment]
![image](https://user-images.githubusercontent.com/23409026/174413533-e9a33a68-1b21-447f-a324-c7392ccb4443.png)
